security/configure-with-vault/README.md

```diff
@@ -172,8 +172,8 @@ vault kv put secret/jksPassword.txt password=jksPassword=mystorepassword
 ```
 
 ```
-cat /tmp/credentials/rbac/mdsPublicKey.pem | base64 | vault kv put /secret/mdsPublicKey.pem mdspublickey=-
```

Comment:
hey @rohit2b I reviewed the changes again and noticed that the final changes will be as follows: 

`cat /tmp/credentials/rbac/mds-publickey.txt | base64 | vault kv put /secret/mdsPublicKey.pem mdspublickey=-
cat /tmp/credentials/rbac/mds-tokenkeypair.txt | base64 | vault kv put /secret/mdsTokenKeyPair.pem mdstokenkeypair=-`